database-restore-n8n.sh

```
@@ -13,162 +14,139 @@ dir="$(dirname "$(realpath "$0")")"
+    for i in "${!date_folders[@]}"; do
+        folder="${date_folders[$i]}"
         folder_name=$(basename "$folder")
-        echo "$count) $folder_name ($backup_count backups)"
-        count=$((count + 1))
+        backup_count=$(find "$folder" -name "*.tar.gz" | wc -l)
+        echo "$((i+1))) $folder_name ($backup_count backups)"
     done
-    
+
     echo ""
     read -p "Select a date folder (number): " folder_choice
```

Shellcheck (0.10.0)

[info] read without -r will mangle backslashes.

(SC2162)